pkg/services/rendering/rendering.go

```diff
@@ -429,6 +433,11 @@ func (rs *RenderingService) getGrafanaCallbackURL(path string) string {
 		return fmt.Sprintf("%s%s&render=1", rs.Cfg.RendererCallbackUrl, path)
 	}
 
+	if rs.Cfg.RendererCallbackUrl != "" {
+		// &render=1 signals to the legacy redirect layer to
+		return fmt.Sprintf("%s%s&render=1", rs.Cfg.RendererCallbackUrl, path)
+	}
```

Comment:
Nit: this can actually be merged with the block above:
```
	if rs.Cfg.RendererUrl != "" || rs.Cfg.RendererCallbackUrl != "" {
		// The backend rendering service can potentially be remote.
		// So we need to use the root_url to ensure the rendering service
		// can reach this Grafana instance.
		
		// &render=1 signals to the legacy redirect layer to
		return fmt.Sprintf("%s%s&render=1", rs.Cfg.RendererCallbackUrl, path)
	}
```

Reply:
whoops thought I addressed this and pushed before merging... but did not. [created minor follow up PR](https://github.com/grafana/grafana/pull/100089) to implement this feedback 👍